rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -185,11 +184,126 @@ local function handle_certs()
   end
 end
 
+local function get_backends_with_name(name)
+  local backends_data = configuration_data:get(name)
+
+  local backends, err = cjson.decode(backends_data)
+  if not backends_data then
```

Comment:
Please check this condition,
we can check `backends_data` first, then check `backends`

Reply:
you are right。 👍

Reply:
@zhaogaolong  It is not correct here.  `backends` not checked